frontend/components/molecules/SidebarButton/index.tsx

```diff
@@ -0,0 +1,76 @@
+type SidebarButtonProps = {
+  IconName: string;
+  Text: string;
+  isSelected: boolean;
+  url: string;
+};
+
+import {
+  BookmarkIcon,
+  QuestionsIcon,
+  RolesIcon,
+  TagsIcon,
+  TeamsIcon,
+  UsersIcon,
+} from "../../public/icons";
+
+import {
+  HiBookmark,
+  HiHome,
+  HiCode,
+  HiUsers,
+  HiOutlineUser,
+  HiUserGroup,
+} from "react-icons/hi";
+
+import Link from "next/link";
+
+const SidebarButton = ({
+  IconName,
+  Text,
+  isSelected,
+  url,
+}: SidebarButtonProps): any => {
+  const getIcon = (): JSX.Element => {
+    if (IconName === "Bookmark") {
+      return <HiBookmark />;
+    }
+    if (IconName === "Questions") {
+      return <HiHome />;
+    }
+    if (IconName === "Roles") {
+      return <HiOutlineUser />;
+    }
+    if (IconName == "Tags") {
+      return <HiCode />;
+    }
+    if (IconName === "Teams") {
+      return <HiUsers />;
+    }
+    if (IconName === "Users") {
+      return <HiUserGroup />;
+    }
+    return <></>;
+  };
```

Comment:
Can you please create atoms for this and refactor using switch case to shorten the code in this molecules